.github/workflows/examples-efr32.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
     if: github.actor != 'restyled-io[bot]'
 
     container:
-      image: ghcr.io/project-chip/chip-build-efr32:47
```

Comment:
cloudbuild also builds efr32. Could we update those as well?

Reply:
thanks for the reminder! updated